docs/web-apps/automated-testing/cypress/yaml/v1.md

```diff
@@ -883,20 +883,6 @@ excludeSpecPattern: ['**/*.*']
 
 ---
 
-### `headless`
```

Comment:
References to `headless` in saucectl configs is framework specific config and unrelated to sauce's `headless` product. So this should not be removed. Same applies to the changes to the v1alpha doc, playwright and testcafe docs.

Reply:
@mhan83 thanks. I've updated to only remove the links to the page for the Sauce Headless product.